### ISSUE TITLE
"all sets in standard" includes not yet released sets

### ISSUE THREAD

**Issue by tooomm** [glacials/whatsinstandard#93]
I feel like upcoming sets are not very well separated from current ones in general.
They are not grayed out enough for example, or the others should be bold etc.

**Comment by glacials** [comment]
Thanks for the feedback! I will see what I can do to make that clearer.

**Comment by glacials** [comment]
Let me know what you think of that :) 
<img width="489" alt="screenshot 2018-08-21 10 40 35" src="https://user-images.githubusercontent.com/438911/44418956-afb8c280-a52e-11e8-80c9-8248d8b7494a.png">

**Comment by tooomm** [comment]
@glacials It kinda looks the same? :)
`Dropping Q4 2020` might be bold, but I don't think that this helps with the issue looking at your screenshot.

When I opened the issue it was mostly about the blue boarder including an all not-included block. But once "Guilds of Ravnica" is out the block would need to be half included in the blue, half out...
I could imagine that it helps if we have a new separate "block" with `Upcoming`, `Entering` or `Joining in X` headline in addition to the `Dropping` ones.

---

Am I the only one thinking that the presentation style isn't ideal and that it's difficult to spot things on a quick look? 😆

**Comment by NilsEnevoldsen** [comment]
I agree that it's not quite as clear as it could be. Here are three things that make me do a double-take:

* As @tooomm says, "All Sets in Standard" graphically encompasses sets that aren't in Standard yet (such as Guilds of Ravnica).
* The color red means "not/wrong/error" to me. Seeing red on a page about what's in Standard makes me instantly assume that these sets are _not_ in Standard. Yellow would be a more appropriate color.
* The word "dropping" has two meanings. "Dropping" can mean "being released". (Example: ["Core Set 2019 drops on July 13."](https://www.polygon.com/2018/7/5/17537636/magic-new-core-set-2019-nicol-bolas-dragon)) I'd prefer "exiting" or "leaving".

**Comment by glacials** [comment]
All good feedback! Thanks folks, sorry for misinterpreting your original feedback @tooomm. Will go back to the drawing board.

**Comment by glacials** [comment]
Toyed around a bit and came up with this, live on the beta site (https://beta.whatsinstandard.com).

Open to more feedback.

<img width="517" alt="screenshot 2018-08-21 17 55 45" src="https://user-images.githubusercontent.com/438911/44436723-794d6880-a56b-11e8-91ae-d9494dbf96b7.png">

As sets release, they will enter the blue box one by one. This is what it will look like when Guilds of Ravnica releases:

<img width="520" alt="screenshot 2018-08-21 18 07 36" src="https://user-images.githubusercontent.com/438911/44437068-32607280-a56d-11e8-9d2e-e6455bf95471.png">

**Comment by NilsEnevoldsen** [comment]
I'm sure there's more than one way to skin the cat, but I like all these changes. Moving the "Exiting" to the bottom is also a good change I didn't think of. As is "Exiting _Standard_".

**Comment by tooomm** [comment]
This is indeed a good improvement @glacials! 👍 
Thanks for taking another look.

<br>

I'm wondering why there are several `...` in the upcoming box still.
Wasn't that to show that this block, all exiting at the same time, has two slots open?

With the new layout more focused on what's currently legal and the split-up of those blocks in sets already in and upcoming ones, i feel like that this "exiting blocks" meaning is less relevant and we can remove those altogether?

**Comment by glacials** [comment]
Ah yeah, I had forgotten about the `...`s! I agree they definitely should be removed in this new layout.

We use [service workers][1], which might cache your page for up to a day. If you want to override this you can force a hard refresh + cache clear. I've also made an issue to see if we can (#98) make sure they update on clients whenever we push a code change.

[1]: https://developers.google.com/web/fundamentals/primers/service-workers/

**Comment by tooomm** [comment]
>I'll see if the space at the bottom is removable, I remember struggling to get it out cleanly with native Bootstrap and giving up.

Result looks very clean!
![spacing](https://user-images.githubusercontent.com/9874850/44539500-9b2e2300-a704-11e8-8820-acdd2947cafe.png)
